library/chips/src/com/android/ex/chips/RecipientEditTextView.java

```diff
@@ -256,6 +257,12 @@ public void onItemClick(AdapterView<?> adapterView,View view, int position,
             }
         };
         setInputType(getInputType() | InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS);
+        setMovementMethod(new ArrowKeyMovementMethod() {
+            @Override
+            public boolean onTouchEvent(TextView widget, Spannable buffer, MotionEvent event) {
+                return event.getAction() != MotionEvent.ACTION_UP && super.onTouchEvent(widget, buffer, event);
+            }
+        });
```

Comment:
The default ArrowKeyMovementMethod behavior causes the selection to move inside the visible region on ACTION_UP if the view was scrolled. RecipientEditTextView maintains the selection at the end all the time, and moving the selection causes the view to jump back to the end after the user finishes scrolling.